### PR TITLE
use popover in dropdown

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

**Comment by Radha-Alberta** [comment]
- [ ] Dropdown is not working as expected in storybook, angular and react

https://www.loom.com/share/0791f1512371459e8ac64c86d53f02a8

https://www.loom.com/share/c433d267846c48b798324ae1051dba34

File: libs/web-components/src/components/popover/Popover.svelte [hunk 1]
```diff
@@ -123,7 +125,8 @@
 
   // Opens the popover and adds the required binding to the new slot element
   function openPopover() {  
-    (async () => {
+    if (_disabled) return;
+    ;(async () => {
```
Comment (vanessatran-ddi) [inline libs/web-components/src/components/popover/Popover.svelte:129]: Is it an extra ; here?

Comment (chrisolsen) [reply libs/web-components/src/components/popover/Popover.svelte:129]: It was on purpose

Comment (vanessatran-ddi) [reply libs/web-components/src/components/popover/Popover.svelte:129]: Thank you Chris! I just read about semicolon before() on google. Good to know!

**Review by vanessatran-ddi — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**